ui/js/component/userVerify/view.jsx

```diff
@@ -26,37 +26,125 @@ class UserVerify extends React.PureComponent {
+            <div className="meta">
+              {__(
+                "A $1 authorization may temporarily appear with your provider."
+              )}{" "}
+              {" "}
+              <Link
+                href="https://lbry.io/faq/identity-requirements"
+                label={__("Read more about why we do this.")}
+              />
+            </div>
+          </div>
+        </section>
+        <section className="card card--form">
+          <div className="card__title-primary">
+            <h3>{__("2) Proof via YouTube")}</h3>
+          </div>
+          <div className="card__content">
+            <p>
+              {__(
+                "If you have a YouTube account with published videos, you can sync your account to be granted instant verification."
+              )}
+            </p>
+          </div>
+          <div className="card__actions">
+            <Link
+              href="https://api.lbry.io/yt/sync"
+              button="alt"
+              icon="icon-youtube"
+              label={__("YouTube Account Sync")}
+            />
+          </div>
+          <div className="card__content">
+            <div className="meta">
+              This will not automatically refresh after approval. Once you have
+              synced your account, just navigate away or click
+              {" "} <Link navigate="/rewards" label="here" />.
+            </div>
+          </div>
+        </section>
+        <section className="card card--form">
+          <div className="card__title-primary">
+            <h3>{__("3) Proof via Chat")}</h3>
+          </div>
+          <div className="card__content">
+            <p>
+              {__(
+                "A moderator capable of approving you is typically available in the #verification channel of our chat room."
+              )}
+            </p>
+            <p>
+              {__(
+                "This process will likely involve providing proof of a stable and established online or real-life identity."
+              )}
+            </p>
+          </div>
+          <div className="card__actions">
+            <Link
+              href="https://slack.lbry.io"
+              button="alt"
+              icon="icon-slack"
+              label={__("Join LBRY Chat")}
+            />
+          </div>
+        </section>
+        <section className="card card--form">
+          <div className="card__title-primary">
+            <h5>{__("Or, Skip It Entirely")}</h5>
+          </div>
+          <div className="card__content">
+
+            <p className="meta">
+              {__(
+                "You can continue without this step, but you will not be eligible to earn rewards."
+              )}
+            </p>
+
+          </div>
+          <div className="card__actions">
+            <Link
+              onClick={() => navigate("/discover")}
+              button="alt"
+              label={__("Skip Rewards")}
+            />
+          </div>
+        </section>
```

Comment:
More of a personal opinion, but I feel verification by chat should be the first option, although I assume this would increase the workload for community moderators. I think requesting for credit / debit card info first will put some people off. Perhaps, credit card or debit card verification should even be the last option.

Reply:
Obviously the real solution is to randomize them and see if it makes a difference :)

I'm going to leave this as-is since it's a smaller change, but I don't feel strongly about the order.